app/dts/behaviors/toggle_layer.dtsi

```diff
@@ -0,0 +1,9 @@
+/ {
```

Comment:
We've not been consistent here, but please add the copyright + license notice to the header here.